cirq-core/cirq/work/work_pool.py

```diff
@@ -1,91 +0,0 @@
-# Copyright 2019 The Cirq Developers
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     https://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-import asyncio
-import collections
-from typing import Optional, Awaitable, Union
-
-
-class CompletionOrderedAsyncWorkPool:
```

Comment:
this seems like something someone may be relying on in cirq user land?

Reply:
It's certainly possible. I can restore this and deprecate it instead.